### PR TITLE
removed ternary and changed fontstyle

### PR DESCRIPTION
**Pull request by mariasykpotter**

### CODE REVIEW

File: client/src/components/MenuPost/style.scss [hunk 1]
```diff
@@ -4,41 +4,38 @@
...
   }
 
 
   .menu-link {
     margin: auto;
     color: $color-black;
-    font-size: 20px;
-    font-family: 'Times New Roman';
+    font-weight: 500;
+    font-size: 14px;
```
Comment (Volodia1999) [inline client/src/components/MenuPost/style.scss]: Use rem for font-size

File: client/src/components/MenuPost/index.tsx [hunk 2]
```diff
@@ -53,21 +52,17 @@ class MenuPost extends React.Component<IProps, IState> {
                     <Row><div className='mt-1'>.</div></Row>
                     <Row><div className='mt-1'>.</div></Row>
                 </Container>
-            {/* TODO Replace undefined in ternary operator with a function for Edit component*/}
-                {menuData.length && (
-                    <nav className={ `navig ${ this.state.isMenuOpen && 'show-menu' }` }>
+                    <nav className={ `navig float-right ${ this.state.isMenuOpen && 'show-menu' }` }>
```
Comment (Volodia1999) [inline client/src/components/MenuPost/index.tsx]: Is it good class name 'navig'?

File: client/src/components/Post/index.tsx [hunk 3]
```diff
@@ -193,26 +195,29 @@ export default class Post extends React.Component<IProps> {
...
                                             <div className='comments ml-lg-0 pl-lg-0 pl-4'>
-                                                <div className='row'>
+                                                <div className='d-flex flex-row justify-content-between'>
                                                     <img
                                                         src={this.props.user.photo || noAvatar}
                                                         alt='avatar'
                                                         width={32}
                                                         height={32}
-                                                        className='img-fluid mt-2 mr-2'
+                                                        // className='img-fluid mt-2 mr-2 justify-self-start'
```
Comment (anna-kuzii) [inline client/src/components/Post/index.tsx]: Comment

File: client/src/components/Post/index.tsx [hunk 4]
```diff
@@ -193,26 +195,29 @@ export default class Post extends React.Component<IProps> {
                                                                         mt-2 mr-2'
                                             />
                                             <span>{this.props.user.username}</span>
-                                            <p className='pl-2'>{this.props.userPosts.selectedPost.description}</p>
+                                            <p className='pl-2 justify-self-start align-self-start'>
+                                            {this.props.userPosts.selectedPost.description}</p>
                                         </div>
```
Comment (anna-kuzii) [inline client/src/components/Post/index.tsx]: You use a lot of this.props. How about destr obj?

**Pull request merged**